commands.py

```diff
@@ -554,20 +526,20 @@ async def poll(self, ctx):
         `!poll end` ends current poll in channel and returns results
         """
 
-        poll_list = ctx.message.content[6:].split(" | ")
+        poll_list = tuple(map(str.strip, ctx.message.content[6:].split("|")))
```

Comment:
I think this just overcomplicates it lmao

and I would rather have users input the command with the proper syntax